app/src/main/java/org/covidwatch/android/CovidWatchApplication.kt

```diff
@@ -95,6 +97,11 @@ class CovidWatchApplication : Application() {
 
     }
 
+    private fun startEnableBluetoothService(){
+        val intent = Intent(this, EnableBluetoothService::class.java)
+        this.startService(intent)
```

Comment:
This is actually starting a regular Service that will run only while the app is running. To start a "foreground service" you need to use a pattern like this:
`if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
			context.startForegroundService(serviceIntent);
		}
		else {
			context.startService(serviceIntent);
		}`
And in the service you have to present a constant notification while the service is running: https://developer.android.com/guide/components/services#StartingAService
In any way Google recommend to use WorkManager for all background tasks as it provides better management across versions and types of tasks (e.g. deferrable tasks are not run at a strict time period and as a result don't consume as much battery): 

> Note: The WorkManager API offers a flexible way of scheduling tasks, and is able to run these jobs as foreground services if needed. In many cases, using WorkManager is preferable to using foreground services directly.